app/models/semester.rb

```diff
@@ -11,6 +11,22 @@ def self.from_year(year, autumn)
     Semester.new(year + (autumn ? '1' : '2'))
   end
 
+  def current?
+    if (2..8).include? Time.current.month
```

Comment:
Could it be possible to make this function more descriptive?
I think we should implement a `self.current` method on the `Semester` class that returns the current semester.
Than this method would just needed to compare them:
`self.to_s == Semester.current.to_s`